librz/analysis/analysis.c

```diff
@@ -144,8 +144,13 @@ RZ_API RzAnalysis *rz_analysis_new(void) {
 
 RZ_API void plugin_fini(RzAnalysis *analysis) {
 	RzAnalysisPlugin *p = analysis->cur;
-	if (p && p->fini && !p->fini(analysis->plugin_data)) {
-		RZ_LOG_ERROR("analysis plugin '%s' failed to terminate.\n", p->name);
+	if (p && p->fini) {
```

Comment:
actually it's easier to have there directly `analysis->plugin_data = NULL;` after the if.
no need to change that if.
Ps: be sure to run `./sys/clang-format.py`

Reply:
Understand ~. Originally I though we can't set it to NULL if this plugin_data wasn't finished correctly.